src/Paket.Core/PackageMetaData.fs

```diff
@@ -93,10 +115,10 @@ let merge templateFile metaData =
     match templateFile with
     | { Contents = ProjectInfo(md, opt) } -> 
         let merged = 
-            { Id = md.Id ++ metaData.Id
-              Version = md.Version ++ metaData.Version
-              Authors = md.Authors ++ metaData.Authors
-              Description = md.Description ++ metaData.Description }
+            { Id = metaData.Id ++ md.Id
```

Comment:
Go for it :+1: . The operator makes more sense the other way round to me - it takes the first value unless it's ``None``